main.js

```diff
@@ -39,11 +39,12 @@ function getGoalCandidates(board) {
 }
 
 function shuffleArray(array) {
-  for (let i = array.length - 1; i > 0; i--) {
+  let shuffled = array.splice()
```

Comment:
Double-check [`splice`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice?v=example) vs [`slice`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice?v=example). Make sure you understand how these unfortunately-named functions work 😄 
Then you are approved to merge.